custom_platforms.js

```diff
@@ -56,13 +56,18 @@ module.exports = {
         "path": "ios",
         "configuration": {
             "removeRulePatterns": [
+                // In Safari, 'if-domain' and 'unless-domain' do not support regexps, only '*'
+                // https://github.com/AdguardTeam/FiltersRegistry/pull/806
+                "\\$domain=\/",
+                ",domain=\/",
+                // Do not exclude scriptlets which contain '$$' when excluding '$$' and '$@$' rules
```

Comment:
нужна ссылка на задачу (тут и ниже)
```suggestion
                // Do not exclude scriptlets which contain '$$' when excluding '$$' and '$@$' rules
                // https://github.com/AdguardTeam/FiltersRegistry/issues/731
```